doc/docportal/advanced_topics/understand_graphics.rst

```diff
@@ -164,4 +186,31 @@ When enabled, ScummVM uses bilinear interpolation instead of nearest neighbor fo
 
 Nearest neighbor is a simple way to scale an image; each pixel becomes multiple pixels of the same color. While this preserves the sharper details in a pixel art image, it also creates "jagged" edges as the image is scaled up. Bilinear interpolation finds the average color between pixel color values and fills in missing pixel, which results in a "smoothed" image.
 
-To toggle between bilinear interpolation and nearest neighbor, press :kbd:`Ctrl+Alt+f`.
\ No newline at end of file
+To toggle between bilinear interpolation and nearest neighbor, press :kbd:`Ctrl+Alt+f`.
+
+3D Graphics settings
+---------------------
+
+Vsync
+*******
+V-sync, or vertical sync, synchronizes the frame rate of the game with the monitor's refresh rate. This helps to prevent screen "tearing", where the screen displays parts of multiple frames at once, which looks like a horizontal line where the image appears to split. 
```

Comment:
..with monitor's or video capture refresh rate.

The latter is more important than the former in the modern world: we have almost no CRTs but Twitch.